compiler/rustc_resolve/src/ident.rs

```diff
@@ -865,7 +877,8 @@ impl<'ra, 'tcx> Resolver<'ra, 'tcx> {
             .find_map(|binding| if binding == ignore_binding { None } else { binding });
 
         if let Some(finalize) = finalize {
-            return self.finalize_module_binding(
+            // `res_mut` is safe here, we have a finalize.
```

Comment:
There are dozens of uses of `res_mut` and you could put a comment like this on every one of them, so it's not very useful.